### DISCUSSION TITLE
Code Complexity

### DISCUSSION THREAD

**Discussion by badar-chachar** [Cyfrin/security-and-auditing-full-course-s23#177] [Q&A] [answered]
I am following the first audit video PasswordStore and what tool is used to find the complexity of the code in the video?

**Comment by EngrPips** [comment, marked as answer]
I think it is `solidity metrics`

**Comment by badar-chachar** [comment]
yeah got it

![image](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/144626994/bf3963fb-dc2b-4a25-aed7-a713f87f6388)

**Reply by EngrPips** [reply]
Interesting. Enjoy!
